FileUtil/FileUpload.py

```
@@ -0,0 +1,45 @@
+        if not os.path.exists(folder):
+            os.makedirs(folder)
+        filename = f"{uuid.uuid4()}_{file.filename}"
+        full_path = os.path.join(folder,filename)
+
+        with open(full_path,"wb") as f:
+            f.write(content)
+
+        return  f"/{full_path}"
+
+    @staticmethod
+    def delete_file(file_path: str):
```

Ruff (0.14.11)

Missing return type annotation for staticmethod `delete_file`

Add return type annotation: `None`

(ANN205)